lib/tiled/sprite.rb

```diff
@@ -18,13 +18,13 @@ def primitive_marker
     end
 
     # @return [Tiled::Sprite] return sprite object from `Tiled::Tile` object.
-    def self.from_tiled(x, y, tile)
+    def self.from_tiled(x, y, tile, w: nil, h: nil)
```

Comment:
I agree, it makes sense to move the `tile` argument to the first place. 

Or it would be even better to replace all arguments with keyword arguments.